client/src/main/java/org/apache/rocketmq/schema/registry/client/serde/json/JsonDeserializer.java

```diff
@@ -73,9 +90,7 @@ public T deserialize(String subject, byte[] payload) {
             jsonNode = objectMapper.readValue(buffer.array(), start, length, JsonNode.class);
```

Comment:
schemaRecordId seems useless in the method, because schema is already obtained by subject.

Is there any plan to add the consistency check between recordID and the obtained schema, or add method to deserialize totally by payload like T deserialize(byte[] payload) ?

Reply:
We considered using the recordID deserialized to check with the obtained schema's. @ferrirW